manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -2311,6 +2311,10 @@ spec:
             description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
+            properties:
+              name:
+                maxLength: 63
```

Comment:
note I don't think this is quite right -- the path is only being applied on the first schema version (per the max_name_size patch's definition). The code check is probably a better tool but it's worth asking ourselves if we still want this to be applied on the schema definition

Reply:
So in this case, the `maxLength` validation is only on `v1alpha` and not `v1beta1` too.

Reply:
Chatted about this offline. Conclusion: we are going to just leave this as for this PR, but will investigate a more general/robust way to validate name lengths for appropriate CRs.

Reply:
Added an issue for this -- https://github.com/open-policy-agent/gatekeeper/issues/3093